node/lib/util/stitch_util.js

```diff
@@ -873,6 +873,8 @@ exports.listCommitsToStitch = co.wrap(function *(repo,
     }
     const commitShas = exports.listCommitsInOrder(commit.id().tostrS(),
                                                   allParents);
+    console.log(`listing all ${commitShas.length} unconverted ancestors of ` +
+                `${commit.id.tostrS()}: ${commitShas}`);
```

Comment:
There could be a number of commits in `commitSha`, let's truncate it if it is too long. 
You can do:
1. for each sha, take only the beginning 7 chars (full length long sha to short-sha format)
2. join array with comma
3. truncate if the total `length` exceeds 1K

Reply:
Good point, will do